### ISSUE TITLE
ESP32 support?

### ISSUE THREAD

**Issue by bam80** [rosmo/go-hidproxy#8]
ESP32-C3/S3 has all the needed to make such proxy.

**Comment by rosmo** [comment]
Yes, that's certainly true. Would require a rewrite though and I don't use this very actively anymore...
